### PR TITLE
chore: Update webview to v 14.0.2

### PR DESCRIPTION
**Pull request by tommasini**
## **Description**

Minor update to update webview to a patch version

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -15713,16 +15713,16 @@ escape-html@^1.0.3, escape-html@~1.0.3:
   resolved "https://registry.yarnpkg.com/escape-html/-/escape-html-1.0.3.tgz#0258eae4d3d0c0974de1c169188ef0051d1d1988"
   integrity sha1-Aljq5NPQwJdN4cFpGI7wBR0dGYg=
 
-escape-string-regexp@2.0.0, escape-string-regexp@^2.0.0:
-  version "2.0.0"
-  resolved "https://registry.yarnpkg.com/escape-string-regexp/-/escape-string-regexp-2.0.0.tgz#a30304e99daa32e23b2fd20f51babd07cffca344"
-  integrity sha512-UpzcLCXolUWcNu5HtVMHYdXJjArjsF9C0aNnquZYY4uW/Vu0miy5YoWvbV345HauVvcAUnpRuhMMcqTcGOY2+w==
-
```
Comment (jpcloureiro) [inline yarn.lock:15720]: Seems unrelated. Can we drop this change?

**Review by jpcloureiro — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (tommasini) [reply yarn.lock]: image
It seems that it's a dependency of the react native webview itself

Comment (jpcloureiro) [reply yarn.lock:15720]: Never mind @tommasini, we [bumped escape-string-regexp since the last release ](https://github.com/MetaMask/react-native-webview-mm/commit/b76b2d6b4c0886c64d5d096e04bfe0ee53783631)

All good here! :shipit:

**Review by jpcloureiro — APPROVED** [review]
shipit
